lib/warnings.js

```diff
@@ -0,0 +1,20 @@
+var chalk = require('chalk')
+
+module.exports = {
+  deprecation: function (template, name) {
+    var initCommand = 'vue init ' + template.replace('-2.0', '') + ' ' + name
+
+    console.log(chalk.red('  This template is deprecated.'))
```

Comment:
Perhaps it would be good to add a bit more information here. For example:

> This template is deprecated, as the original template now uses Vue 2.0 by default.

Just so that they know the new command will still use 2.0.